meson.build

```diff
@@ -145,16 +145,30 @@ if get_option('opencl-kernel-path') != ''
 endif
 
 if get_option('enable-biqgemm')
-  # check if BiQGEMM directory exist. otherwise, throw an error
-  fs = import('fs')
-  if fs.is_dir('../BiQGEMM')
-    extra_defines += '-DENABLE_BIQGEMM=1'
-    biqgemm_inc = include_directories('../BiQGEMM')
+  # In order to enable biqgemm, BiQGEMM lib, which is header-only library, is required.
+  # This meson tries to find its installation.
+  #   1. Checking /usr/include/
+  #   2. Checking path meson-option specifies
+  biqgemm_path = '/usr/include/BiQGEMM'
```

Comment:
Refer to https://mesonbuild.com/Builtin-options.html
```
biggemm_path  = join_paths(prefix, includedir, `BiQGEMM`)
```

If someone cross-compiles nntrainer with biqgemm, this is going to be critical.